src/_base/docker/image/lighthouse/root/app/run.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+reportDir="/home/headless/results"
+
+mkdir -p "${reportDir}"
+
+lighthouse --no-enable-error-reporting \
+           --chrome-flags="--headless --no-sandbox=true" \
+           --output-path=/home/headless/results/lighthouse-results.html \
+           ${TARGET_URL}
```

Comment:
shellcheck is complaining that TARGET_URL needs double quoting.

I'd also add `set -e -o pipefail` to after the shebang line to exit on any line erroring.

Reply:
Improved and re-tested.